internal/provider/bootstrap.go

```diff
@@ -197,8 +198,22 @@ func oneTimeBootstrap(l logging.StandardLogger, c *providerConfig.Config, vpnSet
 	}
 
 	if utils.IsOpenRCBased() {
+		// Instead of creating an empty file, let's use the one generated by the k3s installer
+		content, err := os.ReadFile("/etc/init.d/k3s")
```

Comment:
I think the `svc.WriteUnit()` call below [will simply overwrite the same file](https://github.com/kairos-io/kairos-sdk/blob/eaa9d3ce040b31c27b29751c9a017b9e51c8ec5c/machine/openrc/unit.go#L54). We could as well just leave it as is I guess. It simplifies things though if we just ignore the fact that this is the same path, because we also "Enable()" the svc etc.